requirements.txt

```diff
@@ -0,0 +1,7 @@
+""" 
+zainstalowane pakiety na moim środowisku:
+peewee - ORM dla sqlite
+
+"""
```

Comment:
U mnie to przeszkadzało zainstalować pakiety z pliku requirements ;) musiałme dodać `#` do każdej linii